df_to_azure/export.py

```diff
@@ -32,10 +33,18 @@ def df_to_azure(
     dtypes=None,
     parquet=False,
     clean_staging=True,
+    container_parquet="parquet",
 ):
 
     if parquet:
-        DfToParquet(df=df, tablename=tablename, folder=schema, method=method, id_field=id_field).run()
+        DfToParquet(
+            df=df,
+            tablename=tablename,
+            folder=schema,
+            method=method,
+            id_field=id_field,
+            container_parquet=container_parquet,
```

Comment:
Kunnen we het `container_name` noemen ipv `container_parquet`? Wat denk jij?

Reply:
That's ok, we will alway use containers for parquet, so we do not need to clarify it in the variable name.